src/Umbraco.Web.BackOffice/Controllers/ContentTypeController.cs

```diff
@@ -590,32 +590,41 @@ public ActionResult<ContentTypeImportModel> Upload(List<IFormFile> file)
 
                 var root = _hostingEnvironment.MapPathContentRoot(Constants.SystemDirectories.TempFileUploads);
                 var tempPath = Path.Combine(root,fileName);
-
-                using (var stream = System.IO.File.Create(tempPath))
-                {
-                    formFile.CopyToAsync(stream).GetAwaiter().GetResult();
-                }
-
-                if (ext.InvariantEquals("udt"))
+                if (Path.GetFullPath(tempPath).StartsWith(Path.GetFullPath(root)))
                 {
-                    model.TempFileName = Path.Combine(root, fileName);
+                    using (var stream = System.IO.File.Create(tempPath))
+                    {
+                        formFile.CopyToAsync(stream).GetAwaiter().GetResult();
```

Comment:
The Upload method could be made public async Task<ActionResult<ContentTypeImportModel>>

Reply:
Hey @nzdev that's a good point. However this PR was meant to only add the SECURITY.md file, since the path traversal fixes were merged on #11537 
I don't mind opening a new PR to change this method to public async Task, but it should probably not be part of this one (?)